spec/lib/reports/orders_and_fulfillment/order_cycle_supplier_totals_by_distributor_report_spec.rb

```diff
@@ -23,14 +23,22 @@ module OrdersAndFulfillment
         end
 
         it "generates the report" do
-          expect(report_table.length).to eq(2)
+          expect(report_table.length).to eq(6)
         end
 
         it "has a variant row under the distributor" do
           supplier = order.line_items.first.variant.product.supplier
           expect(report.rows.first.producer).to eq supplier.name
           expect(report.rows.first.hub).to eq distributor.name
         end
+
+        it "lists products sorted by name" do
+          order.line_items[0].variant.product.update(name: "Cucumber")
```

Comment:
I forgot to tell you that it's usually better to use `update!` in specs instead of `update`. The reason is that it raises an exception if something goes wrong and you notice errors earlier. It doesn't really matter in this case because the test would fail with product names like `"Product #123"` and you would immediately guess that the updating didn't work. But in other cases a failed update can cause a lot of confusion.